src/main/java/ch/njol/skript/expressions/ExprARGB.java

```diff
@@ -0,0 +1,78 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Keywords;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Color;
+import ch.njol.util.Kleenean;
+
+import java.util.Locale;
+import java.util.function.Function;
+
+@Name("Alpha/Red/Green/Blue Color Value")
+@Description({
+	"The alpha, red, green, or blue value of colors. Ranges from 0 to 255.",
+	"Alpha represents opacity."
+})
+@Examples({
+	"broadcast red value of rgb(100, 0, 50) # sends '100'",
+	"set {_red} to red's red value + 10"
+})
+@Keywords({"ARGB", "RGB", "color", "colour"})
+@Since("INSERT VERSION")
+public class ExprARGB extends SimplePropertyExpression<Color, Integer> {
+
+	static {
+		register(ExprARGB.class, Integer.class, "(:alpha|:red|:green|:blue) [value]", "colors");
```

Comment:
i'm thinking having value optional might lead to some odd usage

Reply:
I suppose `red's red` isn't great lol
i'll change it

Reply:
what do you think of `component` instead of value? this is what JS uses

Reply:
yeah that could be an alternative